src/containers/ProfileContainer.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useAppDispatch, useAppSelector } from '../hooks'
+import { setClient } from '../state/clientSlice'
+
+const Container = styled.div`
+  width: 50vw;
+  height: 8rem;
+  max-width: 30rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin-top: 3vh;
+  z-index: 1;
+`
+
+const ProfileDiv = styled.div`
+  width: 5em;
+  height: 8rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  & p{
+    font-size: 0.8rem;
+  }
+`
+
+const ProfileName = styled.p<{selected: boolean}>`
+  border-radius: 0.4rem;
+  background-color: rgba(255, 255, 255, ${(props)=>(props.selected? "1":"0")});
```

Comment:
props 대신 destructuring을 사용하시면 더욱 가독성이 좋은 코드가 될것 같습니다.

```suggestion
${({ selected }) => selected ? '1': '0'}
```

Reply:
감사합니다. 참고하도록 하겠습니다.